shift_assignments.md

```diff
@@ -0,0 +1,149 @@
+| `id`                        | `number` | no           | ID                                                             | `1`                                   |
+| `name`                      | `string` | yes          | Name                                                           | `Nachtschicht`                        |
+| `starttime`                 | `string` | no           | Starttime of the shifttype in format `HH:MM`                   | `17:15`                               |
+| `endtime`                   | `string` | no           | Endtime of the shifttype in format `HH:MM`                     | `01:00`                               |
+| `responsability_id`         | `number` | no           | ID of the responsability                                       | `121`                                 |
+| `remark`                    | `string` | yes          | The remark                                                     | `Mit abrechnen`                       |
+| `prefix`                    | `string` | yes          | The abbreviation                                               | `NS`                                  |
+| `color`                     | `string` | yes          | The color                                                      | `#105020`                             |
+| `plannable_category_id`     | `number` | no           | Id of the grouping                                             | `857`                                 |
+| `labels`                    | `array`  | yes          | A list of "Labels", ordered by `technical_name` alphabetically | (See below)                           |
+| `pauses`                    | `array`  | yes          | A list of "Pauses"                                             | (See below)                           |
+| `pauses_duration`           | `number` | no           | Duration of all pauses as decimal number                       | `0.25`                                |
+| `pauses_duration_in_hh_mm`  | `string` | no           | Duration of all pauses in format `HH:MM`                       | `00:15`                               |
+| `working_duration`          | `string` | no           | Duration of the working time as decimal number                 | `8.5`                                 |
+| `working_duration_in_hh_mm` | `string` | no           | Duration of the working time in format `HH:MM`                 | `08:30`                               |
+| `created_at`                | `string` | no           | Datetime in format `yyyy-mm-dd HH:MM:SS.XXXXXXXXX +0000`       | `2021-04-19 12:00:00.000000000 +0000` |
+| `updated_at`                | `string` | no           | Datetime in format `yyyy-mm-dd HH:MM:SS.XXXXXXXXX +0000`       | `2021-04-19 12:00:00.000000000 +0000` |
+
+##### Labels
+
+```json
+[
+  {
+    "name": "Essen, CHF 14",
+    "technical_name": "lunch"
+  }
+]
```

Comment:
I think we can remove the surrounding `Array` here.

Reply:
Hmm, but we also did it like this in working_hours...